src/debug/tester/Kconfig

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+config COMP_TESTER
+	tristate "tester module"
+	default n
+	depends on IPC_MAJOR_4
+	help
+	  Select for loadable module for testing
```

Comment:
Given the very generic name, and a new type of module, I think this warrants a bit more text to describe (i.e. the text you have in the git commti). 

And here "loadable" confuses me a bit. Any module can be loadable, so isn't this just a test module.